src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java

```diff
@@ -537,7 +592,9 @@ private void populateSummaryStatisticsFromResponses(
             Map<String, Integer> numResponses,
             Map<String, Integer> numResponsesExcludingSelf) {
 
+        System.out.println("ALSKDJ");
```

Comment:
You should remove the outputs (it wasn't you who put them there though, but it's still good to do it ^^ )